### PR TITLE
Add methods for comment creation

### PR DESCRIPTION
**Pull request by lm-sousa**
Adds method for creating a comment joinpoint
Adds method for converting any joinpoint into a comment

### CODE REVIEW

**Comment by Sleepy105** [comment]
I have no idea how to add a method to `AstFactory` that makes comment type joinpoints.
Once that is done, it can be integrated into the `ClavaJoinPoints.comment` method to complete this PR.

File: ClavaLaraApi/src-lara-clava/clava/clava/ClavaJoinPoints.lara [hunk 1]
```diff
@@ -483,3 +483,24 @@ ClavaJoinPoints.forStmt = function($init, $condition, $inc, $body) {
...
+
+/**
+ * @param {$joinpoint} $jp
+ *
+ * @return {$joinpoint}
+ * */
+ClavaJoinPoints.makeComment = function($jp) {
+    return $jp.replaceWith(
+        ClavaJoinPoints.comment($jp.code)
+    );
```
Comment (joaobispo) [inline ClavaLaraApi/src-lara-clava/clava/clava/ClavaJoinPoints.lara:505]:
```suggestion
```

Comment (joaobispo) [reply ClavaLaraApi/src-lara-clava/clava/clava/ClavaJoinPoints.lara:505]: I've added a global action $jp.toComment, with overloads .toComment(prefix) and toComment(prefix, suffix).

**Review by joaobispo — CHANGES_REQUESTED** [review]

**Comment by lm-sousa** [comment]
Fixed by adding ```toComment``` method to jointpoints.

**Pull request closed without merging**